Gemfile

```diff
@@ -21,12 +21,15 @@ gem 'shoulda-matchers', '~> 4.0'
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages more meaningful and capitalized 👍 because some of your commit messages are not capitalized 😨 Please consider this issue to make your pull request professional and descriptive ☺️